### PR TITLE
added bogged and armadillo

### PR DESCRIPTION
**Pull request by JuusK**
added bogged and armadillo as pets

### CODE REVIEW

**Comment by Jakllp** [comment]
Oh thank you!  
Have you tested this yet?  
I believe that this might not actually work yet as the mobs aren't being registered yet right?  
They basically need to be added to https://github.com/MyPetORG/MyPet/blob/master/modules/v1_21_R1/src/main/java/de/Keyle/MyPet/compat/v1_21_R1/entity/MyAttributeDefaults.java

**Comment by Jakllp** [comment]
Aaand...  
It would be better if you switch over to the dev branch. On your end and then for the PR.

Other than that: Thank you so much for the contribution already!

File: modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java [hunk 1]
```diff
@@ -35,6 +35,9 @@ public enum MyPetType {
     Allay("ALLAY", "1.19", MyAllay.class, new Compat<>()
             .v("1.19", "allay")
             .search()),
+    Armadillo("ARMADILLO", "1.21", MyArmadillo.class, new Compat<>()
```
Comment (Jakllp) [inline modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java:38]: Weren't they added in 1.20.5/6?

Comment (Jakllp) [reply modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java:38]: But this is fine too - once it's in dev I can easily just copy paste some stuff into v1_20_R4 and change that

Comment (JuusK) [reply modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java:38]: aight i put it in dev branch

Comment (JuusK) [reply modules/API/src/main/java/de/Keyle/MyPet/api/entity/MyPetType.java:38]: i thought it was from 1.21 lmfao

**Comment by Jakllp** [comment]
Well that's not exactly what I meant - it will need to be the dev branch on your end too otherwise this insane merge (see above) happens

**Comment by JuusK** [comment]
how am i supposed to do that?
i dont have a dev branch, would i need to just undo all those commits?

sorry for late response btw

**Comment by Jakllp** [comment]
I went ahead and used a lot of your work in that commit there. Had to fix a few small things.

Thanks for preparing all that.

As for the dev-branch: I think you can somehow tell your fork to have the dev branch as well but I don't know how.  
I might change how the branches for this plugin work soon anyway (aka just have master and build when I say so)

**Comment by JuusK** [comment]
alright, thanks!

**Pull request closed without merging**